packages/react/src/hooks/useWalletWebSocket.ts

```diff
@@ -65,18 +68,26 @@ export function useWalletWebsocket(parameters: UseWalletParameters = {}) {
       topic: WALLET_ROUTE(walletId),
     }
     const symmetricKey = getSymmetricKey(config)
-    const [auth, expiration] = config.utils.build_auth_headers_symmetric(
-      symmetricKey,
+    const headers = addExpiringAuthToHeaders(
+      config,
+      body.topic,
+      {},
```

Comment:
nit: here, and for other places where you add auth headers to websockets, can you:
```suggestion
      {}, // headers
```
to make it clear what this empty object represents?